Framework/PageObjects/Header.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Threading;
+using Framework.Pages;
+using Framework.Pages.ProductsPages;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+
+namespace Framework.PageObjects
+{
+    public class Header
+    {
+        public IWebDriver webDriver;
+
+        public Header(IWebDriver webDriver)
+        {
+            this.webDriver = webDriver;
+        }
+
+        #region Locators
+
+        //********* Menu_1              [line1]
+        //private By MENU_1_BTN = By.XPath("//div[@class='line1']//buttom[@class='mobil']");
```

Comment:
I guess you'll uncomment these locators later? Or they are redundant/wrong?

Reply:
These locators will be used in a future. Especially "By MENU_1_BTN", because it opens hamburger menu on mobile version site.